tests/browser-tests/Login.browser.ts

```diff
@@ -2,7 +2,7 @@ import { DashboardPage, FilesPage, LoginPage, AuthorDetailsPage, DetailsPage, Na
 
 fixture`Getting Started`.page`http://localhost:9000`;
 
-test('assert nav bar', async() => {
+test.only('assert nav bar', async() => {
```

Comment:
this left in by mistake? (not that this file is up to scratch yet anyhow)

Reply:
It needs a better name, but it's literally a test that focus on clicking the nav etc

Reply:
Ohh. opps. Nice catch with the .only

Reply:
Pushed up a fix.